src/Core/GraphConstants.php

```diff
@@ -23,7 +23,7 @@ final class GraphConstants
     const REST_ENDPOINT = "https://graph.microsoft.com/";
 
     // Define HTTP request constants
-    const SDK_VERSION = "1.37.0";
+    const SDK_VERSION = "v2.1.";
```

Comment:
Why this change?

Reply:
Looking into this

Reply:
Fixed.
I created a v2 branch on this repo to merge changes to before the next release.
This branch shows up on packagist as `v2.x-dev`
Issue was caused by the `IncrementMinorVersion.php` script not detecting that a version with the `-dev` suffix is a branch